src/js/components/EditableBlock.js

```diff
@@ -0,0 +1,66 @@
+import Component from '../core/Component.js';
+
+const SPACE_KEY = ' ';
+const BACK_SPACE_KEY = 'Backspace'
+const ENTER_KEY = 'Enter'
+
+const EditableBlock = class extends Component{
+  timer = null;
+  template() {
+    const { index, className, placeholder, text} = this.state
+    return `
+      <div 
+        class="${className} editable-block"
+        contenteditable="true"
+        data-index="${index}" 
+        placeholder='${placeholder}'
+        autofocus
+      >${text}</div>
+    `
+  }
+
+  setEvent() {
+    const { onConvert, onRemove, onEditing, onCreate } = this.props;
+    
+    this.$target.addEventListener('keydown', (e) => {
+      const {target, key} = e
+      const blockIndex = Number(target.dataset.index);
+      const text = target.textContent;
+
+      e.stopImmediatePropagation();
+
+      if (key === SPACE_KEY && text.length < 4) {
+        e.preventDefault();
+        onConvert(blockIndex, text)
+      }
+
+      if (key === BACK_SPACE_KEY && text.length === 0) {
+        onRemove(blockIndex, text)
```

Comment:
```suggestion
        e.preventDefault();
        onRemove(blockIndex, text)
```
여기서도 `e.preventDefault();` 을 넣어줘야 삭제할 때 뒤로가기 되는 것을 막을 수 있을 것 같아요!